public/components/notebooks/components/helpers/default_parser.tsx

```diff
@@ -109,6 +109,8 @@ export const defaultParagraphParser = (defaultBackendParagraphs: any) => {
         visStartTime: vizParams.visStartTime,
         visEndTime: vizParams.visEndTime,
         visSavedObjId: vizParams.visSavedObjId,
+        dataSourceMDSId: paraObject.dataSourceMDSId,
```

Comment:
will older notebooks give an error here? Not sure the usage later but should this be something like: 
```
 dataSourceMDSId: paraObject.dataSourceMDSId ?? '',
```